docs/Product-Guides/Automated-Connect-Flow.md

```diff
@@ -1,4 +1,4 @@
-# Connect
+# Automated Connect Flow
```

Comment:
Ok this is a little tricky because we use ACF to refer to the assisted connect flow.

To disambiguate, instead of "Automated Connect Flow" and "Assisted Connect Flow" maybe we keep "Finch Connect" vs "Assisted Connect Flow". Open to other ideas too. 

What I like about "Finch Connect" is that it makes Automated the default / implied version.

Reply:
See my comment above. 

My proposal: "Finch Connect (its own product) either uses Automated API Connections or Assisted API Connections. Which is why I was proposing renaming them to Automated Connect Flow and Assisted Connect Flow in our docs."

Side note: we should avoid using acronyms when referring to our products. It avoids ambiguity with our customers.

Reply:
> What I like about "Finch Connect" is that it makes Automated the default / implied version.

I can get behind this, but it was not intuitive that this was the case when I was reading the docs for the first time.

Reply:
What is the difference between this [doc](https://developer.tryfinch.com/docs/reference/ZG9jOjMyMDI2ODgw-connect#connect-in-your-users-onboarding)? and this [doc](https://developer.tryfinch.com/docs/reference/ZG9jOjMyMDI2ODg1-overview)? We should either combine them or name them separately.

Reply:
I think the former is intended to be a high level product guide and the latter is meant to be more on a technical level. I agree, not super clear and needs some rethinking.